katas/test/src/pokerhands/jorge/PokerHandsTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace PokerGame.Test.Jorge
+{
+    using PokerGame.Hands.Jorge;
+
+    public class Poker
+    {
+        [Fact]
+        public void WinnerWinner()
+        {
+            //Should win Black with Four of a Kind.
+            Assert.Equal(
+                "Black wins - with Four of a kind",
+                new PokerGameBlackAndWhite().BothHands(
+                    "Black: 2H 2D 2S 2C KD  White: 2C 3H 4S 8C AH"
+                )
+            );
+            Assert.Equal(
+                "White wins - with Straight Flush",
+                new PokerGameBlackAndWhite().BothHands(
+                    "Black: 2H 2D 2S 2C KD  White: AH JH QH TH KH"
+                )
+            );
+            ////Should be tie with tiebreaker of Highcard winning White
```

Comment:
Try to avoid committing commented code unless there is a very good reason to keep it, otherwise, they might polute code